README.md

```diff
@@ -40,13 +32,28 @@ let load = loader({
       return server;
     }
   },
-});
+```
+
+## Loading
+
+The loader components are all handed to this library, which returns a `load`
+function.  Calling this function with a component name will return the result
+of that component's setup function (after recursively setting up any of the
+component's requirements)
 
-// Create server
-let server = await load('server');
+```js
+let loader = require('taskcluster-lib-loader');
+
+// Create loader
+let load = loader({
+  someComponent: { /* ... */ }
```

Comment:
Yeah, I was trying not to reproduce what appeared above in *Components*, but I suppose that full example makes more sense here, with the example in that section trimmed to a single component.